pom.xml

```diff
@@ -567,6 +565,8 @@
                 <module>serialization-tests</module>
                 <module>acceptance-tests</module>
                 <module>performance-tests</module>
+                <module>test-coverage</module>
```

Comment:
should this be squashed with 1c2a0f0d7a26aca1dc957ddc6aed4914e490ccfb ?

Reply:
It's debatable but I think it's thematically different. The modules that rely on Scala don't compile in Java 9, 10, or 11 regardless of other profiles or configuration. Turning on Jacoco does cause Java 11 to fail, but not just on the test-coverage module. So I do think there ought to be a new Jacoco maven profile, but it wouldn't have any automatic activation criteria. And I don't think we run coverage builds in travis, so it's outside my current focus.

Reply:
I am fine it not being squashed. Was just asking!